Headers/DebugServer2/Core/SoftwareBreakpointManager.h

```diff
@@ -29,7 +29,7 @@ class SoftwareBreakpointManager : public BreakpointManager {
   virtual int hit(Target::Thread *thread, Site &site) override;
 
 protected:
-  virtual void getOpcode(uint32_t type, ByteVector &opcode) const;
+  virtual void getOpcode(size_t size, ByteVector &opcode) const;
```

Comment:
just calling out that this is wider than original code on 64-bit platforms. IDK if that's correct, hoping you do :)

Reply:
Right, it is wider than the 32-bit on 64-bit platforms.  However, the one thing that I did miss is that there are 2 additional files that were adjusted and are missing here.  The idea is that this is an Software Breakpoint Manager only interface and that it never was a proper enumerator, it is supposed to mirror the type parameter in the Hardware Breakpoint Manager, which uses an enumeration.  My justification is that it is easier to follow the logic for it to be logically named and used rather than maintaining the uniformity across hardware and software variants.